src/main/java/io/redisearch/Query.java

```diff
@@ -459,4 +461,13 @@ public Query setSortBy(String field, boolean ascending) {
         _sortAsc = ascending;
         return this;
     }
+
+    public Query indexAlias(String alias) {
```

Comment:
@dengliming I don't think we need this API, the all idea behind Alias is that application can work with Alias like they work with index and when a new index is created the alias can be switched without effecting the application

Reply:
Revert